src/main/webapp/resources/css/main.css

```diff
@@ -299,3 +303,13 @@ a.donate-approve {
     margin-right: 30px;
     margin-bottom: 25px;
 }
+
+.img-center {
```

Comment:
bad class name - should express intended use if possible. For example `.project img` or `.project-picture`